include/libp2p/multi/hash_type.hpp

```diff
@@ -17,6 +17,7 @@ namespace libp2p::multi {
     sha1 = 0x11,
     sha256 = 0x12,
     sha512 = 0x13,
+    blake2b_256 = 0xB220,
```

Comment:
```suggestion
    blake2b_256 = 0xb220,
```
just for consistency purposes with the existing code